lib/hooks/queries/useTradeHistory.ts

```diff
@@ -95,7 +96,19 @@ type MarketHeader = {
   categories: { name: string }[];
 };
 
-export const useTradeHistory = (address: string) => {
+export type TradeHistoryItem = {
+  marketId: MarketId;
+  question: string;
+  assetIn: string | undefined;
```

Comment:
Think this is the same and shorter
```suggestion
  assetIn?: string;
```